Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env groovy
+
+/**
+ * Build Switchcounter Workshop
+ *
+ */
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '30')),
+    parameters([
+        stringParam(defaultValue: 'http://downloads.creatordev.io/creator-openwrt/v1/pistachio/marduk/creator-sdk-1.tar.bz2',
+            description: 'Creator image tarball to use', name: "CREATOR_IMAGE"),
+    ])
+])
+
+node('docker && imgtec') {
+    def docker_image
+    docker_image = docker.image "imgtec/workshop-builder:latest"
+    docker_image.inside {
+        stage('Prepare') {
+            checkout([$class: 'GitSCM',
```

Comment:
Do you really need all the extra options in this case?

Instead you could replace all of this with just `checkout scm`, also I would suggest download the sdk first as it just seems a bit more logical as that is the manual process followed to build a package with the SDK?